src/transformers/modeling_flax_utils.py

```diff
@@ -367,6 +372,22 @@ def from_pretrained(
         missing_keys = model.required_params - set(state.keys())
         unexpected_keys = set(state.keys()) - model.required_params
 
+        # Mistmatched keys contains tuples key/shape1/shape2 of weights in the checkpoint that have a shape not
+        # matching the weights in the model.
+        mismatched_keys = []
+        for key in state.keys():
+            if key in random_state and state[key].shape != random_state[key].shape:
+                if ignore_mismatched_sizes:
+                    mismatched_keys.append((key, state[key].shape, random_state[key].shape))
+                    state[key] = random_state[key]
+                else:
+                    raise ValueError(
+                        f"Trying to load the pretrained weight for {key} failed: checkpoint has shape "
+                        f"{state[key].shape} which is incompatible with the model shape ({random_state[key].shape}). "
```

Comment:
```suggestion
                        f"{state[key].shape} which is incompatible with the model shape {random_state[key].shape}. "
```
Think it's better to not add `(` and `)` so that `{state[key].shape}` and `{random_state[key].shape}` are of the same formard to not confuse the reader